keepercommander/commands/enterprise.py

```diff
@@ -236,6 +237,14 @@ def register_command_info(aliases, command_info):
 audit_report_parser.exit = suppress_exit
 
 
+security_audit_report_parser = argparse.ArgumentParser(prog='security-audit-report', description='Run a security audit report.')
+security_audit_report_parser.add_argument('--syntax-help', dest='syntax_help', action='store_true', help='display help')
+security_audit_report_parser.add_argument('--format', dest='format', action='store', choices=['csv', 'json', 'table'], default='csv', help='output format.')
```

Comment:
Still printing in `csv` format because `format` will always be populated and assignment on line 3110 will be overwritten.